### DISCUSSION TITLE
Who uses Less ?

### DISCUSSION THREAD

**Discussion by iChenLei** [less/less.js#3642] [Show and tell]
Hi, folks ! Do you know which open source project or company using Less as css preprocessor ? Welcome show Less user case !

1. open source project name / company name
2. project logo / company logo (in which case it might be used in the Less's README.md / Doc website etc)
3. project github link / comapany official website
4. why you use Less and not sass/stylus ? (not required!)

Thanks !

**Comment by iChenLei** [comment]
> Case Example

1. Ant Design
2. Antd Logo
<img src="https://camo.githubusercontent.com/363242675617648bfbedd1610f89ac28df0f9e1bac8749d83109fafdf8524fff/68747470733a2f2f67772e616c697061796f626a656374732e636f6d2f7a6f732f726d73706f7274616c2f4b4470677667754d704766716148506a6963524b2e737667" width="200px">
3. Project Link

https://github.com/ant-design/ant-design

**Comment by simonguo** [comment]
1. rsuite (React Suite)
2. rsuite logo
![logo](https://user-images.githubusercontent.com/1203827/62457219-e40fa800-b7ac-11e9-912d-3d78ca408692.png) ![logo](https://user-images.githubusercontent.com/1203827/65102389-7be3f100-d9fd-11e9-859e-ae9617ed2f91.png)
3. Project Link 
  - https://github.com/rsuite/rsuite 
  - https://rsuitejs.com/
